pkg/lib/aws/gen_resource_metadata.py

```diff
@@ -126,26 +126,61 @@ def get_nat_metadata(pricing):
 
 
 def get_ebs_metadata(pricing):
-    for product_id, product in pricing["products"].items():
+    storage_mapping = {}
+
+    for _, product in pricing["products"].items():
         if product.get("attributes") is None:
             continue
         if product.get("productFamily") != "Storage":
             continue
-        if product["attributes"].get("volumeApiName") != "gp2":
+        # ignore legacy standard storage
+        if product["attributes"].get("volumeApiName") == "standard":
             continue
 
         price_dimensions = list(pricing["terms"]["OnDemand"][product["sku"]].values())[0][
             "priceDimensions"
         ]
         price = list(price_dimensions.values())[0]["pricePerUnit"]["USD"]
-        return {"price": float(price)}
+
+        metadata = {
+            "type": product["attributes"].get("volumeApiName"),
+            "priceGB": float(price),
+        }
+        # io1 has per IOPS pricing --> add pricing to metadata
+        # if storagedevice does not price per IOPS will set value to 0
+        if product["attributes"].get("volumeApiName") == "io1":
+            # go through pricing data until found data about IOPS pricing
+            for _, product_iops in pricing["products"].items():
+                if product_iops.get("attributes") is None:
+                    continue
+                if product_iops.get("productFamily") != "System Operation":
+                    continue
+
+                price_dimensions = list(pricing["terms"]["OnDemand"][product_iops["sku"]].values())[
+                    0
+                ]["priceDimensions"]
+                price = list(price_dimensions.values())[0]["pricePerUnit"]["USD"]
+
+                # add io1 IOPS pricing to metadata
+                if "io1" == product_iops.get("attributes")["volumeApiName"]:
+                    metadata["priceIOPS"] = price
+                    # add information about iops configurability
+                    iops_configurable = product_iops["attributes"]["provisioned"]
+                    metadata["iops_configurable"] = iops_configurable
```

Comment:
Is there a case when this is "No" for `io1`? If not, what do you think of using a boolean instead of string, and setting it to `True` here?